WordPress/Classes/Networking/PostServiceRemoteXMLRPC.m

```diff
@@ -164,6 +169,54 @@ - (void)deletePost:(RemotePost *)post
     }
 }
 
+- (void)trashPost:(RemotePost *)post
+          forBlog:(Blog *)blog
+          success:(void (^)(RemotePost *))success
+          failure:(void (^)(NSError *))failure
+{
+    NSParameterAssert([post.postID longLongValue] > 0);
+    NSNumber *postID = post.postID;
+    if ([postID longLongValue] > 0) {
+        NSArray *parameters = [blog getXMLRPCArgsWithExtra:postID];
+
+        WPXMLRPCRequest *deletePostRequest = [self.api XMLRPCRequestWithMethod:@"wp.deletePost" parameters:parameters];
```

Comment:
I didn't say we should try and fit everything into 80 characters - only the lines that can be easily broken and in which it makes sense to do so.  The method I commented on has a few examples of this.